### PR TITLE
Cloud user investigation inputs description fix

### PR DESCRIPTION
**Pull request by OmriItzhak**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-8940)

## Description
Updated the outputs description for `cloud user investigation - generic` playbook

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic_README.md [hunk 1]
```diff
@@ -44,42 +44,42 @@ This playbook does not use any commands.
...
+| AwsUserRoleChangesCount | The number of user roles that were changed by the user in the AWS environment. | number |
+| AwsSuspiciousActivitiesCount | The number of suspicious activities performed by the user in the AWS environment. | number |
+| AwsScriptBasedUserAgentCount | The number of script-based user agent usages by the user in the AWS environment. | number |
+| AwsAccessKeyActivitiesCount | The number of access key activities performed by the user in the AWS environment. | number |
+| AwsSecurityChangesCount | The number of security rules that were changed by the user in the AWS environment. | number |
+| AwsAdminActivitiesCount | The number of administrative activities performed by the user in the AWS environment. | number |
+| AwsApiAccessDeniedCount | The number of API accesses denied by the user in the AWS environment. | number |
+| AwsFailedLogonCount | The number of failed logins by the user in the AWS environment. | number |
+| GcpAnomalousNetworkTraffic | Determines whether there are events of anomalous network traffic performed by the user in the GCP environment.<br/>Possible values: True/False. | string |
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic_README.md:56]: Should be boolean

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic_README.md [hunk 2]
```diff
@@ -44,42 +44,42 @@ This playbook does not use any commands.
...
+| GcpSuspiciousApiUsage | Determines whether there are events of suspicious API usage by the user in the GCP environment.<br/>Possible values: True/False. | string |
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic_README.md:57]: Should be boolean

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic_README.md [hunk 3]
```diff
@@ -44,42 +44,42 @@ This playbook does not use any commands.
...
+| GcpFailLogonCount | The number of failed logins by the user in the GCP environment. | number |
+| GsuiteFailLogonCount | The number of failed logins by the user in the G Suite environment. | number |
+| GsuiteUnusualLoginAllowedCount | The number of unusual logins performed by the user and allowed in the G Suite environment. | number |
+| GsuiteUnusualLoginBlockedCount | The number of unusual logins performed by the user and blocked in the G Suite environment. | number |
+| GsuiteSuspiciousLoginCount | The number of suspicious logins performed by the user in the G Suite environment. | number |
+| GsuiteUserPasswordLeaked | Determines whether user's password was leaked in the G Suite environment.<br/>Possible values: True/False. | string |
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic_README.md:63]: Should be boolean

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation_README.md [hunk 4]
```diff
@@ -43,14 +43,14 @@ This playbook does not use any integrations.
...
-| GcpSuspiciousApiUsage | Determines whether there are events of suspicious API usage by the user in the GCP environment. | unknown |
-| GcpFailLogonCount | The number of failed logins by the user in the GCP environment. | unknown |
-| GsuiteFailLogonCount | The number of failed logins by the user in the G Suite environment. | unknown |
-| GsuiteUnusualLoginAllowedCount | The number of unusual logins performed by the user and allowed in the G Suite environment. | unknown |
-| GsuiteUnusualLoginBlockedCount | The number of unusual logins performed by the user and blocked in the G Suite environment. | unknown |
-| GsuiteSuspiciousLoginCount | The number of suspicious logons performed by the user in the G Suite environment. | unknown |
-| GsuiteUserPasswordLeaked | Determines whether the user's password was leaked in the G Suite environment. | unknown |
+| GcpAnomalousNetworkTraffic | Determines whether there are events of anomalous network traffic performed by the user in the GCP environment.<br/>Possible values: True/False. | string |
```
Comment (idovandijk) [inline Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation_README.md:46]: Should be boolean

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation_README.md [hunk 5]
```diff
@@ -43,14 +43,14 @@ This playbook does not use any integrations.
...
+| GcpSuspiciousApiUsage | Determines whether there are events of suspicious API usage by the user in the GCP environment.<br/>Possible values: True/False. | string |
```
Comment (idovandijk) [inline Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation_README.md:47]: Should be boolean

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation_README.md [hunk 6]
```diff
@@ -43,14 +43,14 @@ This playbook does not use any integrations.
...
+| GcpFailLogonCount | The number of failed logins by the user in the GCP environment. | number |
+| GsuiteFailLogonCount | The number of failed logins by the user in the G Suite environment. | number |
+| GsuiteUnusualLoginAllowedCount | The number of unusual logins performed by the user and allowed in the G Suite environment. | number |
+| GsuiteUnusualLoginBlockedCount | The number of unusual logins performed by the user and blocked in the G Suite environment. | number |
+| GsuiteSuspiciousLoginCount | The number of suspicious logons performed by the user in the G Suite environment. | number |
+| GsuiteUserPasswordLeaked | Determines whether the user's password was leaked in the G Suite environment.<br/>Possible values: True/False. | string |
```
Comment (idovandijk) [inline Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation_README.md:53]: Should be boolean

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation.yml [hunk 7]
```diff
@@ -1171,29 +1171,35 @@ inputs:
   playbookInputQuery:
 outputs:
 - contextPath: GcpAnomalousNetworkTraffic
-  description: Determines whether there are events of anomalous network traffic performed by the user in the GCP environment.
-  type: unknown
+  description: |-
+    Determines whether there are events of anomalous network traffic performed by the user in the GCP environment.
+    Possible values: True/False.
+  type: string
```
Comment (idovandijk) [inline Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation.yml:1177]: Should be boolean

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation.yml [hunk 8]
```diff
@@ -1171,29 +1171,35 @@ inputs:
...
 - contextPath: GcpSuspiciousApiUsage
-  description: Determines whether there are events of suspicious API usage by the user in the GCP environment.
-  type: unknown
+  description: |-
+    Determines whether there are events of suspicious API usage by the user in the GCP environment.
+    Possible values: True/False.
+  type: string
```
Comment (idovandijk) [inline Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation.yml:1182]: Should be boolean

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation.yml [hunk 9]
```diff
@@ -1171,29 +1171,35 @@ inputs:
...
   description: The number of suspicious logons performed by the user in the G Suite environment.
-  type: unknown
+  type: number
 - contextPath: GsuiteUserPasswordLeaked
-  description: Determines whether the user's password was leaked in the G Suite environment.
-  type: unknown
+  description: |-
+    Determines whether the user's password was leaked in the G Suite environment.
+    Possible values: True/False.
+  type: string
```
Comment (idovandijk) [inline Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation.yml:1202]: Should be boolean

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic.yml [hunk 10]
```diff
@@ -532,112 +532,118 @@ inputs:
...
   description: The number of suspicious logins performed by the user in the G Suite environment.
-  type: unknown
+  type: number
 - contextPath: GsuiteUserPasswordLeaked
-  description: Determines whether user's password was leaked in the G Suite environment.
-  type: unknown
+  description: |-
+    Determines whether user's password was leaked in the G Suite environment.
+    Possible values: True/False.
+  type: string
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic.yml:589]: Should be boolean

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic.yml [hunk 11]
```diff
@@ -532,112 +532,118 @@ inputs:
...
+    Determines whether there are events of anomalous network traffic performed by the user in the GCP environment.
+    Possible values: True/False.
+  type: string
 - contextPath: GcpSuspiciousApiUsage
-  description: Determines whether there are events of suspicious API usage by the user in the GCP environment.
-  type: unknown
+  description: |-
+    Determines whether there are events of suspicious API usage by the user in the GCP environment.
+    Possible values: True/False.
+  type: string
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic.yml:569]: Should be boolean

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic.yml [hunk 12]
```diff
@@ -532,112 +532,118 @@ inputs:
...
   description: The number of failed logins by the user in the AWS environment.
-  type: unknown
+  type: number
 - contextPath: GcpAnomalousNetworkTraffic
-  description: Determines whether there are events of anomalous network traffic performed by the user in the GCP environment.
-  type: unknown
+  description: |-
+    Determines whether there are events of anomalous network traffic performed by the user in the GCP environment.
+    Possible values: True/False.
+  type: string
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic.yml:564]: Should be boolean

**Review by idovandijk — CHANGES_REQUESTED** [review]
Looks good. I commented on some outputs that should be Boolean instead of String. Please update them and regenerate playbook README, then ping me and I'll approve :)

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic_README.md (hunk 1 above)
Comment (OmriItzhak) [reply Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic_README.md]: Done

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic_README.md (hunk 2 above)
Comment (OmriItzhak) [reply Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic_README.md]: Done

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation_README.md (hunk 4 above)
Comment (OmriItzhak) [reply Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation_README.md]: Done

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation_README.md (hunk 5 above)
Comment (OmriItzhak) [reply Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation_README.md]: Done

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation.yml (hunk 7 above)
Comment (OmriItzhak) [reply Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation.yml]: Done

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation.yml (hunk 9 above)
Comment (OmriItzhak) [reply Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation.yml]: Done

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic.yml (hunk 10 above)
Comment (OmriItzhak) [reply Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic.yml]: Done

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic.yml (hunk 12 above)
Comment (OmriItzhak) [reply Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic.yml]: Done

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic.yml (hunk 11 above)
Comment (OmriItzhak) [reply Packs/CommonPlaybooks/Playbooks/playbook-Cloud_User_Investigation_-_Generic.yml]: Done

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation.yml (hunk 8 above)
Comment (OmriItzhak) [reply Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation.yml]: Done

File: Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation_README.md (hunk 6 above)
Comment (OmriItzhak) [reply Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation_README.md]: Done

File: Packs/AWS-Enrichment-Remediation/Playbooks/playbook-AWS_-_User_Investigation_README.md [hunk 13]
```diff
@@ -20,17 +20,17 @@ This playbook does not use any sub-playbooks.
 
 ### Integrations
 
-AWS - CloudTrail
+* AWS - CloudTrail
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Enrichment-Remediation/Playbooks/playbook-AWS_-_User_Investigation_README.md:23]:
```suggestion
AWS - CloudTrail
```
If there is only one item in a "list" then that item does not get a bullet.

File: Packs/AWS-Enrichment-Remediation/Playbooks/playbook-AWS_-_User_Investigation_README.md [hunk 14]
```diff
@@ -20,17 +20,17 @@ This playbook does not use any sub-playbooks.
...
 
+* Set
 * LoadJSON
 * GetTime
-* Set
 
 ### Commands
 
-aws-cloudtrail-lookup-events
+* aws-cloudtrail-lookup-events
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Enrichment-Remediation/Playbooks/playbook-AWS_-_User_Investigation_README.md:33]:
```suggestion
aws-cloudtrail-lookup-events
```

File: Packs/Azure-Enrichment-Remediation/Playbooks/playbook-Azure_-_User_Investigation_README.md [hunk 15]
```diff
@@ -20,7 +20,7 @@ This playbook does not use any sub-playbooks.
 
 ### Integrations
 
-Azure Log Analytics
+* Azure Log Analytics
```
Comment (ShirleyDenkberg) [inline Packs/Azure-Enrichment-Remediation/Playbooks/playbook-Azure_-_User_Investigation_README.md:23]:
```suggestion
Azure Log Analytics
```

File: Packs/Azure-Enrichment-Remediation/Playbooks/playbook-Azure_-_User_Investigation_README.md [hunk 16]
```diff
@@ -29,7 +29,7 @@ Azure Log Analytics
 
 ### Commands
 
-azure-log-analytics-execute-query
+* azure-log-analytics-execute-query
```
Comment (ShirleyDenkberg) [inline Packs/Azure-Enrichment-Remediation/Playbooks/playbook-Azure_-_User_Investigation_README.md:32]:
```suggestion
azure-log-analytics-execute-query
```

**Comment by ShirleyDenkberg** [comment]
@idovandijk Doc review completed.

**Review by idovandijk — APPROVED** [review]
great

**Pull request merged**